test/test_helper.rb

```diff
@@ -0,0 +1,6 @@
+require "simplecov"
+SimpleCov.start
+require "minitest/autorun"
+require "mocha/minitest"
+require "minitest/pride"
+require "CSV"
```

Comment:
we can add the pry require here rather than individual files for convenience.

Reply:
agreed